CHANGELOG.md

```diff
@@ -21,6 +19,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Breaking Changes
 
+## dynamic_bonding_curve [0.1.8] [PR #148](https://github.com/MeteoraAg/dynamic-bonding-curve/pull/148)
+
+### Added
+
+- `create_config_for_dammv2_migration` endpoint to allow create `PoolConfig` account for only DAMM v2 migration with LP vesting configuration.
+- `PoolConfig` account now stores `creator_lp_vesting_info` and `partner_lp_vesting_info` fields. Only applicable to DAMM v2 migration option. It store vesting parameters required for `lock_position` cpi during DAMM v2 migration.
+
+### Removed
+
+- Removed `protocol_fee_percentage` and `referral_fee_percentage` fields from `PoolFeesConfig` field from `PoolConfig` account. Will be using defined constant `PROTOCOL_FEE_PERCENTAGE` and `HOST_FEE_PERCENTAGE` as replacement.
```

Comment:
could you make sure that wont be a breaking change in trading terminals? (ex: Jupiter)

Reply:
Updated the changelog to breaking change. Will ask dann to notify integrators about it.